.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
- [ ] Currently, the JavaScript ESLint linter is not installed in your project. It's a requirement that you have the JavaScript ESLint Linter set up so that your JavaSfript code is checked for syntax and correct code structure. Kindly install that so it reflects on GitHub 👍🏽

These are the ones you currently have

<img width="1046" alt="Screenshot 2023-01-31 at 11 24 21 AM" src="https://user-images.githubusercontent.com/50941074/215749651-e5221fa3-57c0-457e-a9ee-ec14ec432e26.png">

Linters / ESLint should be part of the list 

I suggest you reinstall the JavaScript and HTML & CSS linter config as that's what is required for this project 👍🏽